apiserver/common/upgradeseries.go

```diff
@@ -27,8 +27,8 @@ type UpgradeSeriesMachine interface {
 //go:generate mockgen -package mocks -destination mocks/mock_unit.go github.com/juju/juju/apiserver/common UpgradeSeriesUnit
 type UpgradeSeriesUnit interface {
 	AssignedMachineId() (string, error)
-	UpgradeSeriesStatus() (model.UnitSeriesUpgradeStatus, error)
-	SetUpgradeSeriesStatus(status model.UnitSeriesUpgradeStatus) error
+	UpgradeSeriesStatus(model.UpgradeSeriesStatusType) (model.UnitSeriesUpgradeStatus, error)
+	SetUpgradeSeriesStatus(model.UnitSeriesUpgradeStatus, model.UpgradeSeriesStatusType) error
```

Comment:
I would tend toward ordering the args as type then status, but this is arguably pedantry.

Reply:
These args are very likely to go away once the type changes. There will be no need for them since we are moving to one set of states rather than cycling through an abbreviated set twice.